src/biome/text/data/readers.py

```diff
@@ -15,8 +15,8 @@
 PATH_COLUMN_NAME = "path"
 
 __LOGGER = logging.getLogger(__name__)
-# TODO: The idea is to make the readers a class and define a metaclass that they have to follow.
-#       For now, all reader methods have to return a dask.DataFrame. See ElasticsearchDataFrameReader
+# Make the readers a class and define a metaclass that they have to follow.
+# For now, all reader methods have to return a dask.DataFrame. See ElasticsearchDataFrameReader
```

Comment:
I left this comment a long time ago, i think we can just remove it, since i am not sure if going for a meta class here is the right approach!